modules/release-notes/con-relnotes-known-issues.adoc

```diff
@@ -9,16 +9,16 @@ This section lists known issues with {product} {product-version}:
 Deprecation warning: The backend.auth.keys config has been replaced by backend.auth.externalAccess"::
 +
 --
-Due to changes to the backend service-to-service authentication, you will see a deprecation warning if you have set up your deployment to use the "backend.auth.keys" configured under "upstream.backstage.appConfig". This warning is harmless for now and no action is needed. link:{LinkRHIDPIssue}RHIDP-2716[(RHIDP-2716)]
+Due to changes to the backend service-to-service authentication, you will see a deprecation warning if you have set up your deployment to use the "backend.auth.keys" configured under "upstream.backstage.appConfig". This warning is harmless for now and no action is needed. 
+//link:{LinkRHIDPIssue}RHIDP-2716[(RHIDP-2716)]
 --
 
-// TODO https://issues.redhat.com/browse/RHIDP-2766 - to be confirmed by Kim Tsao before including in the KI list
-
-// IP addresses may appear in Application logs::
-// +
-// --
-// There is the potential for IP addresses to appear in application audit logs if RHDH is deployed without a proxy. If there are concerns about the presence of IP addresses in application logs, take the following action:
+IP addresses may appear in application logs::
++
+--
+If you deploy {product-short} without a proxy, there is the potential for user's IP addresses to appear in the audit logs. If your organization's security policies restrict you from retaining IP addresses, you can take one or both of the following actions:
 
-// 1) Delete them from the areas where they are being collected.
-// 2) To avoid further collection of PII in logs, it is recommended that you set up a reverse proxy in front of RHDH so the IP of the proxy is logged instead. link:{LinkRHIDPIssue}RHIDP-2766[(RHIDP-2766)]
-// --
+* Delete the IP addresses after they are collected.
+* Prevent collection of user's IP addresses by configuring a reverse proxy in front of {product-short}.
```

Comment:
```suggestion
* Prevent collection of users' IP addresses by configuring a reverse proxy in front of {product-short}.
```

Reply:
Same as above, maybe just remove users, I like that suggestion